test/flop_phoenix_test.exs

```diff
@@ -1928,6 +1948,42 @@ defmodule Flop.PhoenixTest do
       assert Jason.decode!(phx_click) == [["push", %{"event" => "sort"}]]
     end
 
+    test "application of custom sort directions per column" do
+      html =
+        render_table(
+          [
+            meta: %Flop.Meta{
+              flop: %Flop{
+                order_by: [:ttfb],
+                order_directions: [:desc_nulls_last]
+              }
+            },
+            items: [
+              %{
+                ttfb: 2
+              },
+              %{
+                ttfb: 1
+              },
+              %{
+                ttfb: nil
+              }
+            ],
+            ttfb_directions: {:asc_nulls_last, :desc_nulls_last}
+          ],
+          &test_table_with_custom_sort_directions/1
+        )
+
+      [ttfb_sort_href] =
+        html
+        |> Floki.find("thead th a:fl-contains('TTFB')")
+        |> Floki.attribute("href")
+
+      # assert href representing opposite direction of initial table sort
+      assert ttfb_sort_href =~
+               "order_by[]=ttfb&order_directions[]=asc_nulls_last"
```

Comment:
Comparing the query parameters as a string is a bit volatile, since the order is not necessarily deterministic across Elixir and library versions. Could you decode the query with `Plug.Conn.Query.decode` here?

Reply:
addressed in d0ecffce3574bb900d72bb6e16d5c769bf570126